src/atoms/taxiLocations.ts

```diff
@@ -1,6 +1,8 @@
+import { Location } from "types/location";
```

Comment:
```suggestion
import type { Location } from "types/location";
```

`type only import` 를 할 수 있어요 😄 

https://velog.io/@pandati0710/typescript-type-only-importexport

Reply:
수정했습니다!